### PR TITLE
Update videodevicecontroller_hdrvideocontrol.md

### PR DESCRIPTION
**Pull request by strongstrongyiwen**
Updated hdrvideocontrol doc, indicated that user need to stop streaming before setting value to the control.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@strongstrongyiwen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: windows.media.devices/videodevicecontroller_hdrvideocontrol.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ public Windows.Media.Devices.HdrVideoControl HdrVideoControl { get; }
 Gets the [HdrVideoControl](hdrvideocontrol.md) associated with the [VideoDeviceController](videodevicecontroller.md). This allows you to enable and disable High Dynamic Range (HDR) video capture mode on devices that support it.
 
 ## -property-value
-The [HdrVideoControl](hdrvideocontrol.md) associated with the [VideoDeviceController](videodevicecontroller.md).
+This control requires you to stop streaming before setting value, see [KSPROPERTY_CAMERACONTROL_EXTENDED_VIDEOHDR
```
Comment (kasumman) [inline windows.media.devices/videodevicecontroller_hdrvideocontrol.md]: >you [](http://example.com/codeflow?start=22&length=3)

I might phrase this again without using 'you' word.
"This control requires that the stream is at stopped state before the value is set" or something. (Please note I'm not native english speaker) #Resolved

**Comment by PRMerger17** [comment]
@strongstrongyiwen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by absubba — APPROVED** [review]

**Review by kasumman — APPROVED** [review]
:shipit:

**Comment by PRMerger6** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@drewbatgit Could you review the proposed update? If you approve, we can merge. Thanks.

**Comment by drewbatgit** [comment]
#sign-off

**Review by drewbatgit — APPROVED** [review]

**Pull request merged**